### PR TITLE
feat/airdrop-salvation

### PR DESCRIPTION
**Pull request by taikiFurusyo**
This PR is for users who submitted salvation form to take a chance of the airdrop again.

The target users are:
- Got scammed, but missed sending a form to recover it
- Put the address incorrectly
- Changed discord id and entered address in salvation form

### CODE REVIEW

File: genesis-operation/process-airdrop-operation.ts [hunk 1]
```diff
@@ -16,7 +16,11 @@ const airdropFractionAccountAdd = "ununifi16ll32qkvwr0vz4u7dt4qzvkz0v7nspecd5jae
 const discord_id = 0
 const wallet_id = 1
 const discordMsgCacheFile =  "discord_msg_cache_file.json"
+const useSalvationList = false;
```
Comment (mkXultra) [inline genesis-operation/process-airdrop-operation.ts:19]: change false to true.

File: genesis-operation/process-airdrop-operation.ts [hunk 2]
```diff
@@ -40,12 +44,119 @@ function arrangeAirdropApplication(FileName: string): Array<any> {
   // parse the content of the file
   const records = parse(data);
 
-  // array to have each discord-id and address pair  
-  let arrangedList = records.map((record: any) => [record[2], record[4]]);  
+  // array to have each discord-id and address pair [discord-id, address, lineNum]
+  let arrangedList = records.map((record: any, index: number) => {
+    // index += 2;
```
Comment (mkXultra) [inline genesis-operation/process-airdrop-operation.ts:49]: Delete it

**Review by mkXultra — CHANGES_REQUESTED** [review]

**Comment by mkXultra** [comment]
@taikiFurusyo 
Is this PR a work in progress?

**Comment by taikiFurusyo** [comment]
> @taikiFurusyo Is this PR a work in progress?

I forgot pushing.
Please review, @mkXultra .

**Pull request merged**